### DISCUSSION TITLE
line with basic controls

### DISCUSSION THREAD

**Discussion by tgoeminne** [fabricjs/fabric.js#9662] [General]
Hi,

how do you add a line with 2 controlpoints to edit the line?

Now I add line it shows basic boundingbox and controlpoints, which dont work to edit a line.

**Comment by ShaMan123** [comment]
Use a Polyline
```
const poly = new Polyline(points)
poly.controls = createPolyControls(2)
```

**Comment by tgoeminne** [comment]
this doesnt answer the question. How do you create a canvas your answer also createCanvas(1)?
